CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## X.X.X - 2022-XX-XX
+
+### PaymentSheet
+
+* [ADDED][5422](https://github.com/stripe/stripe-android/pull/5422) Card expiration dates with a single-digit month are now preserved correctly when closing and re-opening the `PaymentSheet` via the `FlowController`.
```

Comment:
I'd say this is a [FIXED]

Reply:
Oops, little copy/paste situation. Fixed now.